lib/galaxy/model/__init__.py

```diff
@@ -222,8 +222,8 @@ def metrics(self):
         return self.text_metrics + self.numeric_metrics
 
     def set_streams(self, stdout, stderr):
-        stdout = galaxy.util.unicodify(stdout)
-        stderr = galaxy.util.unicodify(stderr)
+        stdout = galaxy.util.unicodify(stdout) or u''
```

Comment:
Might read cleaner to set default parameters on the function, any reason not to do that?

Reply:
`unicodify()` returns `None` for `None`, not for `''`

Reply:
@nsoranzo Got it, yeah.  I just realized my suggestion probably didn't make sense, but that's what I was getting at -- instead of having the logic in consumers like this it might be good to standardize behavior at a single point.  That's for future refactoring though, I guess, no worries about it here.